Stripe/PublicHeaders/STPSourceParams.h

```diff
@@ -208,6 +208,65 @@ NS_ASSUME_NONNULL_BEGIN
                                     country:(NSString *)country
                         statementDescriptor:(nullable NSString *)statementDescriptor;
 
+/**
+ Creates params for a Klarna source.
+ @see https://stripe.com/docs/sources/klarna#create-source
+
+ @param returnURL            The URL the customer should be redirected to after
+                             they have successfully verified the payment.
+ @param currency             The currency the payment is being created in.
+ @param purchaseCountry      The ISO-3166 2-letter country code of the customer's location.
+ @param items                An array of STPKlarnaLineItems. Klarna will present these on the confirmation
+                             page. The total amount charged will be a sum of the `totalAmount` of each of these items.
+ @param customPaymentMethods Required for customers located in the US. This determines whether Pay Later and/or Slice It
+                             is offered to a US customer.
+ @param address              An STPAddress for the customer. At a minimum, an email, line1, postal code, city, and country must be provided.
```

Comment:
`billingAddress`

Reply:
Should we mention that `STPAddress#name` will be ignored?

Reply:
The compiler enforces that these docstrings match the name of the parameter, not the name of the selector.

And yeah, we should!

Reply:
thanks for clarifying